server/src/services/report.service.ts

```
@@ -1,35 +1,62 @@
 import { PersonalAllDetailsModel } from '../models/PersonalAllDetails.Model';
 import { ReportModel } from '../models/report.model';
+import validator from 'validator';
+
+function sanitizeNumber(input: unknown): number {
+  const inputStr = String(input);
+  const sanitized = validator.stripLow(validator.whitelist(inputStr, '0-9'));
+  if (!validator.isInt(sanitized)) {
+    throw new Error('Invalid number input');
+  }
+  return parseInt(sanitized, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)